Notely/lib/screens/settings.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:flutter/material.dart';
+import 'package:settings_ui/settings_ui.dart'; //Flutter Settings Package
+//Added under 'dependencies' in pubspec.yaml
+
+List<Icon> icons = [
+  Icon(Icons.person),
+  Icon(Icons.settings),
+  Icon(Icons.credit_card),
+];
+
+Size buttonSize;
+Offset buttonPosition;
+bool isMenuOpen = false;
+
+class SettingsPage extends StatelessWidget {
```

Comment:
This needs to be switched to a stateful widget so that the toggle in the menu actually works properly.